app/src/main/java/ch/epfl/sweng/SDP/game/VotingPageActivity.java

```diff
@@ -344,10 +399,8 @@ public void onComplete(@NonNull Task<byte[]> task) {
                                             // Display the first drawing
                                             String playerName = playersNames[0];
                                             changeDrawing(drawings[0], playerName);
-
```

Comment:
These white lines are needed between code blocks, while the one above after else { not really